wp-content/themes/core/pcss/utilities/mixins/_typography.pcss

```diff
@@ -25,26 +25,134 @@
  * ------------------------------------------------------------------------- */
 
 @define-mixin body-content {
-	font-size: var(--font-size-content-s);
+	line-height: var(--line-height-base);
+	font-size: var(--font-size-body-s);
 
 	@media (--viewport-medium) {
-		font-size: var(--font-size-content);
+		font-size: var(--font-size-body);
 	}
 }
 
+@define-mixin body-content-large {
+	line-height: var(--line-height-xl);
+	font-size: var(--font-size-body);
+
+	@media (--viewport-medium) {
+		font-size: var(--font-size-body-l);
+	}
+}
+
+/* -------------------------------------------------------------------------
+ *
+ * Overline
+ *
+ * ------------------------------------------------------------------------- */
+
+@define-mixin overline {
+	font-family: var(--font-family-calibre);
+	font-size: 14px;
+	font-weight: var(--font-weight-regular);
+	line-height: var(--line-height-base);
+	letter-spacing: 1.75px;
+	text-transform: uppercase;
+}
+
 /* -----------------------------------------------------------------------------
  *
  * Quotes & Blockquotes
  *
  * ----------------------------------------------------------------------------- */
 
 @define-mixin quote-base {
-	border-top: 1px solid var(--color-border);
-	border-bottom: 1px solid var(--color-border);
-	padding: var(--spacer);
+	max-width: 800px;
+	padding: var(--spacer-2);
 }
 
 @define-mixin quote-text {
+	font-family: var(--font-family-calibre);
+	font-size: 32px;
+	line-height: 1.222;
+	font-style: normal;
+	font-weight: var(--font-weight-semibold);
+	text-align: center;
+
+	@media (--viewport-medium) {
+		font-size: 36px;
+	}
+}
+
+/* -----------------------------------------------------------------------------
```

Comment:
For consistency, and same as we've done for the other vars, let's update these mixins to be a 0 based index of sizing from small (0) to large (11). I'll have design update their files as well (we've done this on the TEC views refresh and it's been pretty helpful, removes the trouble with naming things). We can also go ahead and include the body and caption mixins and port them over into here for consistency 💃